src/internal/m365/service/groups/backup.go

```diff
@@ -93,7 +95,8 @@ func ProduceBackupCollections(
 					fb)
 
 				// Backup is always best effort and we don't error out even if this fails
-				operations.LogFaultErrors(ctx, fb.Errors(), "collecting metadata")
+				helpers.LogFaultErrors(ctx, fb.Errors(), "collecting site metadata")
+				logger.CtxErr(ctx, err).Info("collecting site metadata")
```

Comment:
very nitpicky: but this way the info log kinda appears like a header to the error list.
```suggestion
				logger.CtxErr(ctx, err).Info("collecting site metadata")
				helpers.LogFaultErrors(ctx, fb.Errors(), "collecting site metadata")
```